### ISSUE TITLE
Solution to: review threads are hard to follow

### ISSUE THREAD

**Issue by fregante** [sindresorhus/refined-github#3462]
With review threads, often the last comment on the page is not actually the last comment.

How can this be solved?

---

I'm thinking that we could add a link at the bottom of the conversation pointing to the last review comment. Mockup:

<img width="772" alt="" src="https://user-images.githubusercontent.com/1402241/90143827-fa642280-dd75-11ea-95ef-28526ae68b6c.png">

This would be similar to `highlight-highest-rated-comment`, except it would appear at the very bottom of the page, which is where people expect the latest change.

**Comment by Geeyun-JY3** [comment]
I come up with another feature request relating to this solution:
Change the update information`updated <time> ago/ on <date>` to `updated <time> ago/ on <date> by ***` in issue lists if sorted by update date, and make it clickable, linking to "Latest comment" bubble you have mentioned above.
(Would it be a kind of duplicate feature of the `open-issue-to-latest-comment`🤔? Though I still hope this feature can be added.)

**Comment by fregante** [comment]
Makes sense, can you open a new issue? It would still be part of `open-issue-to-latest-comment`

**Comment by Geeyun-JY3** [comment]
> Makes sense, it would still be part of `open-issue-to-latest-comment`
> 
> Actually this feature could be changed to **point to the latest visible change on the page**, instead of **"the bottom-most comment"** and would be a partial solution to this issue.

That’s great!

**Comment by yakov116** [comment]
> Actually this feature could be changed to **point to the latest visible change on the page**

I happen to be 👎 on this change. This would be a "breaking change" I really like the current behavior and I use it all the time to jump to the last comment.

**Comment by fregante** [comment]
Isn't the last comment possibly in a review that's _not_ at the bottom though?

**Comment by Geeyun-JY3** [comment]
> I happen to be 👎 on this change. This would be a "breaking change" I really like the current behavior and I use it all the time to jump to the last comment.

Or just pointing to "Latest comment" bubble which is at the bottom of the conversation would be more acceptable, just like [what I have said](https://github.com/sindresorhus/refined-github/issues/3462#issuecomment-674944698)?  In this case , jumping to the latest comment just need one more click on the bottom.

**Comment by yakov116** [comment]
> Isn't the last comment possibly in a review that's _not_ at the bottom though?

thats if you are looking at a pull request. for issues many tines I am looking to see a solution and check the last 2 comments is what I am looking for.

The truth is that I still miss https://github.com/sindresorhus/refined-github/pull/1879

**Comment by notlmn** [comment]
I usually end up on a PR review from notifications, which most of the times links directly to a specific comment (which is supposedly the "last edited comment"). In the linear flow, I don't this feature making any difference except for a fancy button that points to the "last edited comment".

IMO PRs are supposed to be non-linear, in terms of comments, but I think that there should be view of activities (updates) on PR in linear order somewhere.

**Comment by yakov116** [comment]
I think https://github.com/sindresorhus/refined-github/pull/4029 would help for this too

**Comment by fregante** [comment]
Indeed #4029 helps see the content that matters 🎉

**Comment by Geeyun-JY3** [comment]
@fregante So, how can I find the latest visible change on the page by using the `conversation-activity-filter`? TBH I don't get that😂.

Also, should I create a new separate issue for https://github.com/sindresorhus/refined-github/issues/3462#issuecomment-674944698?
